### ISSUE TITLE
Questions about ChannelAttention

### ISSUE THREAD

**Issue by akaniklaus** [luuuyi/CBAM.PyTorch#3]
Thank you very much! In the paper, this block meant to be the same as SE block (except for the MaxPool addition). Is there any reason why you used nn.Conv2d rather than nn.Linear in the original SE block? I am working with time series, would you recommend me to use Conv1d or keep using Linear? Finally, is there any reason why bias = False? I obtained better results with bias = True with SE block in my case.

Lastly, have you done any comparison with Gather & Excite? https://github.com/hujie-frank/GENet

**Comment by luuuyi** [comment]
First, when the tensor is squeezed to NxCx1x1, nn.Conv2d(kernel_size=1) is same as nn.Linear. I'm not sure nn.Conv2d or nn.Linear work same in time series task.
Second, some papers and researchs said when you put Batchnorm after conv layer in one network, the bias in Conv layer is not necessary, but you can also set bias=True.

**Comment by akaniklaus** [comment]
@luuuyi Thanks for your fast response!

I actually have batch-norms after my convolutional layers but it still made a difference. In the other official repository of BAM and CBAM (attention-module), they have also used nn.Linear with Bias.

**Comment by akaniklaus** [comment]
This is the official code by the way, there seem to be implementation differences also in spatial attention.
https://github.com/Jongchan/attention-module/blob/master/MODELS/cbam.py

**Comment by luuuyi** [comment]
@akaniklaus  Maybe something is different, when I try to implement this work, the official verison is still not release, so I finish my version depend on my understanding of this paper.

**Comment by akaniklaus** [comment]
@luuuyi Yes, I assumed so. I shared with you in case you are not aware of it. So that you can check it.

**Comment by luuuyi** [comment]
@akaniklaus OK, thank your for your suggestion.
